components/infobox/wikis/valorant/infobox_team_custom.lua

```diff
@@ -0,0 +1,65 @@
+---
+-- @Liquipedia
+-- wiki=valorant
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Earnings = require('Module:Earnings')
+local Variables = require('Module:Variables')
+local String = require('Module:String')
+local Template = require('Module:Template')
+
+local TeamRanking = require('Module:TeamRanking')
+
+local CustomTeam = {}
+
+local _team
+
+function CustomTeam.run(frame)
```

Comment:
indents in his function are bad

Reply:
Check now, for whatever reason Github changed my default tab size to 8, and it only effected certain lines

Reply:
Half are spaces, half are tabs

Reply:
Actually, goes for all code not just this function

Reply:
Alright, just made an attempted fix for that